index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Ayoub Chahir | Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <ul>
+        <li id="logo"><a href="#" class="logo">My Logo</a></li>
+        <li><img src="./img/sandwich.svg" alt="Menu Button" /></li>
+      </ul>
+    </header>
+    <div class="background-shapes">
+      <img src="./img/Ellipse-3.svg" class="ellipse-3" alt="" /><img
+        src="./img/Ellipse-4.svg"
+        class="ellipse-4"
+        alt=""
+      /><img src="./img/Ellipse-5.svg" class="ellipse-5" alt="" /><img
+        src="./img/Ellipse-8.svg"
+        class="ellipse-8"
+        alt=""
+      /><img src="./img/Ellipse-11.svg" class="ellipse-11" alt="" /><img
+        src="./img/Ellipse-12.svg"
+        class="ellipse-12"
+        alt=""
+      /><img src="./img/Ellipse-13.svg" class="ellipse-13" alt="" /><img
+        src="./img/Ellipse-14.svg"
+        class="ellipse-14"
+        alt=""
+      /><img src="./img/Ellipse-15.svg" class="ellipse-15" alt="" /><img
+        src="./img/Rectangle-4.svg"
+        class="rect-4"
+        alt=""
+      /><img src="./img/Rectangle-5.svg" class="rect-5" alt="" /><img
+        src="./img/Rectangle-6.svg"
+        class="rect-6"
+        alt=""
+      /><img src="./img/Rectangle-7.svg" class="rect-7" alt="" />
+    </div>
+    <section class="headline">
+      <h1>
+        I'm Ayoub <br />
+        Glad to see you!
+      </h1>
+      <p class="presentation">
+        I'm a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don't hesitate to contact me.
+      </p>
+      <p class="connect">Let's connect</p>
+      <div class="social-media">
+        <img src="./img/twitter.svg" alt="" class="flex-item" /><img
+          src="./img/Linkedin-icon.svg"
+          alt=""
+          class="flex-item"
+        /><img src="./img/medium.svg" alt="" class="flex-item" /><img
+          src="./img/github.svg"
+          alt=""
+          class="flex-item"
+        /><img src="./img/angellist.svg" alt="" class="flex-item" />
+      </div>
```

Comment:
- [ ] you did well but I think you can still improve your code by using semantic elements. I suggest you wrap the whole element in a `ul` tag . Please, make sure that each of your icon is also wrapped in an anchor `a` element which is wrapped in `li` element.
example
```
<ul class="social-media">
<li><img src="./img/twitter.svg" alt="" class="flex-item" /></li>
</ul>
```

Reply:
I just changed them. Thank's